taskcluster/fxci_config_taskgraph/transforms/firefoxci_artifact.py

```diff
@@ -42,7 +42,7 @@ def make_firefoxci_artifact_tasks(config, tasks):
                 # Add private artifacts
                 if "MOZ_FETCHES" in task_def["payload"].get("env", {}):
                     fetches = json.loads(
-                        task_def["payload"].get("env", {}).get("MOZ_FETCHES", {})
+                        task_def["payload"].get("env", {}).get("MOZ_FETCHES", "{}")
```

Comment:
super nitty: could also do a check `if env := task_def["payload"].get("env"):....` for arguably more performance. But give this only runs once, it should be fine too.